R/whoami.R

```diff
@@ -273,8 +274,15 @@ lookup_gh_username <- function(email, token) {
   url <- URLencode(paste0(gh_url, "/search/users?q=", email,
                           " in:email"))
 
+  if(is.null(token)){
+    tokens <- Sys.getenv(c("GITHUB_PAT", "GITHUB_TOKEN"))
```

Comment:
I would  just  do  `token <- Sys.getenv("GITHUB_TOKEN", Sys.getenv("GITHUB_PAT"))` and  then no other changes  are needed.

Reply:
I don't understand (yet) how the lines below use only one of them when both exist? E.g I have the two, one token for gh and one for when I use the V4 API bc I was once told to create two.

Reply:
I am not sure what you mean. With this `GITHUB_TOKEN` is  used,  if set, otherwise `GITHUB_PAT` is used, if set.  Otherwise no token is used. 

I would keep `GITHUB_TOKEN` the first, because that is the default currently.